code/src/components/AddTask.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { useDispatch } from "react-redux"
+
+import { tasks } from 'reducers/Tasks'
+
+import './AddTask.css'
+
+export const AddTask = () => {
+  const [taskInput, SetTaskInput] = useState('')
+  const dispatch = useDispatch()
+  const date = new Date()
```

Comment:
I realised that you could simplify this by just typing `new Date().getTime()` directly on line 16 instead